chihuahua/assetlist.json

```diff
@@ -101,6 +101,42 @@
       },
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "Chihuahua coin",
+      "extended_description": "Chihuahua.baby! Chihuahuas may be small, but they possess a mighty spirit and strong energy. These little dogs are known for their boldness. They are confident, sassy, and full of spunk. They are not afraid to take on much larger adversaries when their loved ones are at stake. Chihuahuas thrive on human companionship and enjoy cuddling up with their owners, providing an abundance of love and affection! Woof woof woof...",
+      "denom_units": [
+        {
+          "denom": "factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/Chihuahua",
+          "exponent": 0
+        },
+        {
+          "denom": "Chihuahua",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/Chihuahua",
+      "name": "Chihuahua coin",
+      "display": "chihuahua.baby",
```

Comment:
this denomination isn't defined in the denom_units.

Reply:
It's still not defined in denom units. The denom unit has a capital 'c' while this reference does not

Reply:
> It's still not defined in denom units. The denom unit has a capital 'c' while this reference does not

"denom_units": [
        {
          "denom": "factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/Chihuahua",
          "exponent": 0
        },
        {
          "denom": "Chihuahua",
          "exponent": 6
        }
      ],
      "base": "factory/chihuahua1mzcfzh4ufk2cta59pm9a6wdyvv8c4v5epqzj46/Chihuahua",
      "name": "Chihuahua",
      "display": "Chihuahua",
      "symbol": "CHIHUAHUA",